rest/src/market_data_rest_kraken_spot.py

```diff
@@ -13,7 +13,7 @@
     to_date
 from core.src.instrument_types import SPOT
 from core.src.markets import KRAKEN
-from core.src.syms import split_currency_pair_into_lhs_rhs, BTC
+from core.src.spot_syms import split_currency_pair_into_lhs_rhs, BTC, check_currency_pair_spot
 from rest.src.market_data_rest import MarketDataRestApi
 from rest.src.request_types import POST
 
```

Comment:
Try to only return variables, instead of converted/truncated objects in the return statement, in prevision of a future logging system.

Reply:
Adressed ✅